cartographer_grpc/proto/cartographer_service.proto

```diff
@@ -21,8 +21,9 @@ import "google/protobuf/empty.proto";
 package cartographer_grpc.proto;
 
 message AddTrajectoryRequest {
```

Comment:
Sorry to start the discussion with this unrelated change.
Should the service be called Cartographer, or, MapBuilder, which is closer to the public API?

Reply:
As discussed offline: renaming to `MapBuilder` service sounds right to me. I will rename the related classes accordingly, i.e. `MapBuilderServer` and `MapBuilderServerOptions`.